src/components/styles/common.scss

```
@@ -7,12 +7,10 @@
   }
 }
 
-.co-spin {
-  animation: co-spin 2s infinite linear;
-}
+.co-spin-pipelines {
```

I'd avoid the `co-` prefix altogether in your plugin. Then you never risk conflicting with console classes that always use the prefix. Something like

```suggestion
.pipelines-console-plugin__spin {
```

would be better imo. See the recommendations here:

https://docs.openshift.com/container-platform/4.17/web_console/dynamic-plugin/overview-dynamic-plugin.html#general-plug-in-guidelines

---

Updated the PR. PTAL.